ui/src/app/layout/topics/topicmetadatatable/topic-metadata-table.component.html

```diff
@@ -49,6 +49,30 @@
                 </ng-template>
             </td>
         </tr>
+        <tr>
+            <th scope="row">{{ 'Producers' | translate }}</th>
+            <td>
+                <ul *ngIf="mappedProducers?.length">
+                    <li *ngFor="let producer of mappedProducers ">
+                        <a *ngIf="producer.infoUrl; else infoUrlTextSub"
+                           [href]="producer.infoUrl"
+                           target="_blank">{{ producer.name }}</a>
+                        <ng-template #infoUrlTextSub>
+                            <span>{{ producer.name }}</span>
+                        </ng-template>
+                        <span *ngIf="isOwnerOfTopic && !(selectedEnvironment | async)?.stagingOnly" class="ml-3">
+                            <button class="btn btn-sm btn-danger" appSpinnerWhile #del="appSpinnerWhile"
+                                    (click)="openDeleteProducerDlg(producer,deleteProducerDlg)">
+                                <i class="fas fa-eraser mr-1"></i>{{ 'Delete Producer' | translate }}
+                            </button>
+                        </span>
+                    </li>
+                </ul>
+                <span *ngIf="!topic.producers?.length"
+                ><i>{{ 'None' | translate }}</i></span
```

Comment:
I think I'd prefer that the whole line (the `tr`) is not shown if there are no additional producers.